src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -39,6 +42,7 @@ type PaginatedData = {
 // Query poll interval
 const POLL_INTERVAL_MS = 10000 // 10 sec
 const FIRST_PAGE_TIMESTAMP = 0
+const LOGGER_TAG = 'transactions/feed/TransactionFeedV2'
```

Comment:
nit: can just call this `TAG`, since it's only ever used for logging purposes anyways (and matches the naming convention we typically use)

Reply:
@jophish sure! I'll address this in the follow-up PR.

Reply:
nit: what's the problem with doing it here, since it's this PR that introduces it?

Reply:
@jeanregisser no problem with doing it here. I thought this should be a small enough change that it can be addressed within the next PR that I was already working on so there's less potential back-and-forth with rebasing all the follow-up PRs. But you're right, it makes sense to make the change here if this change is purely related to this PRs goal.

Reply:
@jeanregisser updated